dotcom-rendering/src/web/components/RichLink.stories.tsx

```diff
@@ -776,3 +777,53 @@ export const PaidFor = () => {
 		</Section>
 	);
 };
+
+export const Analysis = () => {
+	return (
+		<div
+			css={css`
+				/* stylelint-disable-next-line color-no-hex */
+				background-color: #fff4f2;
+			`}
+		>
+			Analysis Articles have a different color background, so rich links
+			should too.
+			<Section
+				showTopBorder={false}
+				centralBorder="full"
+				showSideBorders={false}
+			>
+				<Figure
+					format={{
+						display: ArticleDisplay.Standard,
+						design: ArticleDesign.Analysis,
+						theme: ArticlePillar.News,
+					}}
+					isMainMedia={false}
+					role="richLink"
+				>
+					<RichLink
+						richLinkIndex={1}
+						cardStyle="news"
+						imageData={someImageData}
+						headlineText="Rich link headline"
+						contentType="article"
+						url=""
+						linkFormat={{
+							display: ArticleDisplay.Standard,
+							design: ArticleDesign.Analysis,
+							theme: ArticlePillar.Culture,
+						}}
+						format={{
+							display: ArticleDisplay.Standard,
+							design: ArticleDesign.Analysis,
+							theme: ArticlePillar.Culture,
+						}}
```

Comment:
Do you think it may be better to create a few format variables at the top of the file and use them instead of repeating the formats?

```
const standardFormat = {
					display: ArticleDisplay.Standard,
					design: ArticleDesign.Standard,
					theme: ArticlePillar.News,
				}
```